src/selection/Selection.js

```diff
@@ -52,18 +52,96 @@ define(['EventEmitter'], function (EventEmitter) {
         });
     };
 
+    /**
+     * Clears the selection.
+     */
     Selection.prototype.clear = function () {
         this.selected = [];
-        this.emit('change');
+        this.emit('change', this.selected);
     };
 
     Selection.prototype.primary = function () {
         return this.selected[this.selected.length - 1];
     };
 
-    Selection.prototype.all = function () {
+    Selection.prototype.get = function () {
         return this.selected;
     };
 
+    /**
+     * Selects the selectable object and emits the 'change' event.
+     *
+     * @param {object} selectable an object with element and context properties
+     */
+    Selection.prototype.select = function (selectable) {
+        if (!Array.isArray(selectable)) {
+            selectable = [selectable];
+        }
+
+        if (this.selected[0] && this.selected[0].element) {
```

Comment:
From a separation of concerns perspective, I'd suggest moving these element interactions out of `Selection` and into a separate unit that observes change events and handles those class changes.

Reply:
I agree with @larkin to keep the element interactions in the same unit.